discord_connect_gateway.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * This code was used to make our Discord application connect to a gateway at
+ * least once, as required by Discord before being able to send any message to
+ * a channel.
```

Comment:
Maybe move this file to ./bin ? You should also document what users needs to do with it?

Reply:
Yep, why not.
`node discord_connect_gateway.js`, wait to receive a READY from discord and that's all